common/lib/xmodule/xmodule/js/src/problem/edit.coffee

```diff
@@ -42,8 +48,19 @@ class @MarkdownEditingDescriptor extends XModule.Descriptor
     lineNumbers: true
     lineWrapping: true
     })
+
     if text
       @xml_editor.setValue(text)
+
+    totalLines = @xml_editor.lineCount()
+    totalChars = @xml_editor.getTextArea().value.length
+    @xml_editor.autoFormatRange {
```

Comment:
I'm seeing on google that `autoFormatRange' is deprecated with recent versions, i think we're using `3.21`, can you also confirm that our version support autoFormatRange or not. 
One other thing i noticed that we're using different version in js tests ~ `5.xx` and the tests are failing since it's deprecated.


https://groups.google.com/forum/?fromgroups#!searchin/codemirror/how$20to$20manage$20formatting/codemirror/hdEWC83r94c/PcaVazYL9PAJ